Tasks/AzureAppConfigurationImportV10/task.json

```diff
@@ -5,11 +5,11 @@
     "description": "Import key-values to an Azure App Configuration instance",
```

Comment:
Can we do the similar updates for import and snapshot tasks?

Reply:
How does "Import key-values to an Azure App Configuration instance in your build or deployment pipelines" sound ?